app/views.py

```diff
@@ -0,0 +1,149 @@
+import random
+from datetime import timedelta
+
+from django.contrib.auth.decorators import login_required
+from django.db.models.functions import Length
+from django.http import HttpRequest, HttpResponse, HttpResponseServerError
+from django.shortcuts import get_object_or_404, redirect, render, reverse
+from django.utils import timezone
+from django.views.decorators.csrf import csrf_exempt
+from django.views.generic import View
+
+from api.models import Submission, Transcription
+from app.permissions import require_coc, require_reddit_auth
+from utils.mixins import CSRFExemptMixin
+from website.helpers import get_additional_context
+
+
+@login_required
+@csrf_exempt
+def accept_coc(request: HttpRequest) -> HttpResponse:
+    """Show the Code of Conduct and ask the volunteer to accept it."""
+    if request.method == "POST":
+        request.user.accepted_coc = True
+        request.user.save()
+        return redirect(reverse("choose_transcription") + "?show_tutorial=1")
+    else:
+        return render(request, "app/accept_coc.html", get_additional_context())
+
+
+@login_required
+@require_coc
+@require_reddit_auth
+def choose_transcription(request: HttpRequest) -> HttpResponse:
+    """Provide a user with transcriptions to choose from."""
+    # time_delay = timezone.now() - timedelta(hours=settings.ARCHIVIST_DELAY_TIME)
+    time_delay = timezone.now() - timedelta(hours=60)
```

Comment:
We should probably pick one of these and delete the other

Reply:
I need it for testing -- I'm working against a db export from prod, so since I don't have any new submissions coming in then I have to keep moving the time back so that I keep getting submissions. It will be removed :+1: